README.md

```diff
@@ -206,7 +209,16 @@ cosmos/my-key/1/unsigned.json
 
 To overwrite the first tx, use `--force`.
 
-### List
+### tx vote
+
+```
+multisig tx vote <chain name> <key name> <proposal number> <vote option> [flags]
+```
+
+This will generate a tx for a governance proposal vote and it will push it to s3 directly. You will need to specify the proposal number and the vote (e.g. yes, no). 
+You will also need to specify the denom for the fees (e.g. uatom) if it cannot be retrieved from a node.
```

Comment:
Ideally denom can become part of the config slash gotten from the chain registry so we never have to worry about it like this

Reply:
yes, chain registry would be cool. Created an issue for config update.